crates/router/src/connection.rs

```diff
@@ -280,6 +280,11 @@ impl<
         self.last_packet = Instant::now();
 
         let msg = match item {
+            Err(ProtocolError::Io(e)) if e.kind() == std::io::ErrorKind::ConnectionReset => {
+                log::debug!("[{:?}] connection closed", self.conn_info);
```

Comment:
I do not see how this fixes errors, since it is only about changing log message and log level. Below arm of the match has exactly the same logic